products/bash/setup-demos.sh

```diff
@@ -392,43 +406,45 @@ for row in $(echo "${REQUIRED_ADDONS_JSON}" | jq -r '.[] | select(.enabled == tr
   postgres)
     echo -e "$info [INFO] Releasing postgres...\n"
     if ! $SCRIPT_DIR/release-psql.sh -n $NAMESPACE; then
-      $DEBUG && echo -e "\n$cross [ERROR] Failed to release PostgreSQL in the '$NAMESPACE' namespace"
       update_conditions "Failed to release PostgreSQL in the '$NAMESPACE' namespace" "Releasing"
       update_phase "Failed"
     else
-      $DEBUG && echo -e "\n$tick [SUCCESS] Successfully released PostgresSQL in the '$NAMESPACE' namespace"
+      echo -e "\n$tick [SUCCESS] Successfully released PostgresSQL in the '$NAMESPACE' namespace"
     fi # release-psql.sh
+
+    check_phase_and_exit_on_failed
```

Comment:
This looks like it will stop installing other addons if postgres fails. Could this `check_phase_and_exit_on_failed` be moved to outside the for loop?

Reply:
Changed it to check and exit only at the end of all installations.